source/gui/settingsDialogs.py

```diff
@@ -3132,10 +3132,22 @@ def makeSettings(self, settingsSizer: wx.BoxSizer) -> None:
 		index = [x.value for x in AddonsAutomaticUpdate].index(config.conf["addonStore"]["automaticUpdates"])
 		self.automaticUpdatesComboBox.SetSelection(index)
 
+		# Translators: This is the label for a text box in the add-on store settings dialog.
+		addonMetadataMirrorLabel = _("Metadata &mirror")
```

Comment:
```suggestion
		addonMetadataMirrorLabel = _("Server &mirror URL")
```

Reply:
Thoughts on this wording? (note you may need to expand hidden comments, or check the unresolved discussions).

Reply:
Bumping this comment